app/db/models/index.js

```diff
@@ -28,14 +28,15 @@ if (config.has('db.sequelize.url')) {
   })
 }
 
+// reads in files where we define other models
```

Comment:
I wonder if this file could be simplified using 
https://github.com/sequelize/express-example/blob/master/express-main-example/sequelize/index.js
as a guide. not a big deal, tho

Reply:
Certainly - it could help readability. Some of the complexity of this code is that it reads it all the other JS files in this directory to import - so if it's basically doing what `requireIndex` is for, we can try to replace that code with `requireIndex`. Then have the rest of the model setup follow the example they've provided.